<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>CABEÇALHO IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 16pt &quot;Arial&quot;;
	background-color: #32a041;
	color: rgb(255, 255, 255);
	border: 2px solid #32a041;
	border-radius:20px;
}


QPushButton:hover{
	background-color: rgb(31, 100, 41);
	border: 2px solid  rgb(31, 100, 41);
}</string>
    </property>
    <property name="text">
     <string>CONFIRMAR CABEÇALHO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Versão</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>IHM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Tipo de Serviço</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Tamanho Total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Identificação</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Flags</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Deslocamento de Fragmento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Checksum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Tempo de Vida</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Protocolo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Endereço de Origem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
}

QPushButton:hover{
	
	background-color: rgb(208, 208, 208);

}</string>
    </property>
    <property name="text">
     <string>Endereço de Destino</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
